egs/fisher_swbd/s5/local/chain/run_tdnn_opgru_1b.sh

```diff
@@ -0,0 +1,296 @@
+#!/bin/bash
+# Copyright 2017 University of Chinese Academy of Sciences (UCAS) Gaofeng Cheng
```

Comment:
why do you never have any comparisons between different experiments?  That's what the compare_wer_general.sh script is for.  It would allow us to see what the point of the experiments is.
You have 1a and 1b scripts for OPGRU.  There is no comment here saying how 1b differs from 1a.